platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java

```diff
@@ -299,12 +299,28 @@ public Date getExpires(Subject subject) {
      */
     public SecurityManager getSecurityManager() {
         BundleContext context = getBundleContext();
+
         if (context != null) {
             ServiceReference securityManagerRef =
                     context.getServiceReference(SecurityManager.class);
-            return (SecurityManager) context.getService(securityManagerRef);
+
+            int tries = 0;
```

Comment:
❗️ We should use https://github.com/jhalterman/failsafe instead of hand-coding our own retry logic.